Makefile

```diff
@@ -52,14 +52,18 @@ endif
 PACKAGE_TYPE ?= deb
 
 bin/bazel:
+ifeq (,$(wildcard bin/bazel))
```

Comment:
we don't need to manually check existence for this file: `make` will treat the target as a filename automatically and do it for us

Reply:
But seems on my local macOS, `make clean` would always download the file even if already exists.

Reply:
Tried again, seems we can remove this.